bluemira/equilibria/find.py

```diff
@@ -893,13 +890,7 @@ def grid_2d_contour(x, z):
         z_new.append(zi)
         if not np.isclose(xi, x[i + 1]) and not np.isclose(zi, z[i + 1]):
             # Add an intermediate point (ccw)
-            if x[i + 1] > xi and z[i + 1] < zi:
-                x_new.append(x[i + 1])
-                z_new.append(zi)
-            elif x[i + 1] > xi and z[i + 1] > zi:
-                x_new.append(xi)
-                z_new.append(z[i + 1])
-            elif x[i + 1] < xi and z[i + 1] > zi:
+            if (x[i + 1] > xi and z[i + 1] < zi) or (x[i + 1] < xi and z[i + 1] > zi):
                 x_new.append(x[i + 1])
```

Comment:
This doesn't look right. append z[i+1] to z_new in one case

Reply:
Its a direct copy of what was there as far as I can tell? 
The first half of the if is L896 and the second half is l902. 
L899 is absorbed into the else

Reply:
ah right true; well I guess it is unit-tested anyway! Not sure I find this easier to read TBH but it's all good then